### DISCUSSION TITLE
I would like to be able to set this to "Speed Up by: 0.5x"

### DISCUSSION THREAD

**Discussion by revbiz** [iina/iina#5171] [Q&A]
Hi,
Nice app! I would like to be able to set the "Speed Up by: 2.0 " like the "Speed Down by: 0.5x" so that the result will be  "Speed Up by: 0.5x" also. Is there a way to update this?

Is there also a setting for "Speed Up by"?

I see the related docs here.
https://github.com/iina/iina/discussions/3919

Thanks Dave

**Comment by low-batt** [comment]
If you just want to change the speed to something other than the values provided by the `Playback` menu click on `Show Video Panel` under the `Video` menu. A side panel will appear with the `VIDEO` tab selected. In the `Speed` section there is a slider that can be used to change the speed. There is also a text field where you can directly enter the speed.

If you do this frequently then you can assign a key to do it. In IINA's settings on the `Key Bindings` tab you can press the `Duplicate` button to create a copy of the IINA bindings that you can modify. Add a new key binding to `multiply speed 0.5`.

Issue #3919 is about setting a default speed so that IINA doesn't start playing at 1x. That doesn't sound like what you want to do.
